source/Ship.cpp

```diff
@@ -3864,6 +3865,12 @@ int Ship::StepDestroyed(vector<Visual> &visuals, list<shared_ptr<Flotsam>> &flot
 			for(shared_ptr<Flotsam> &it : jettisoned)
 				it->Place(*this);
 			flotsam.splice(flotsam.end(), jettisoned);
+			for(auto &it : jettisonedFromBay)
+			{
+				it.first->Place(*this, it.second);
+				flotsam.emplace_back(it.first);
+			}
```

Comment:
```suggestion
			for(const auto &[newFlotsam, amount] : jettisonedFromBay)
			{
				newFlotsam->Place(*this, amount);
				flotsam.emplace_back(newFlotsam);
			}
```
Something like this, maybe?

Reply:
No `const`, to allow the compiler to use move semantics when emplacing into the `flotsam` collection. We should actually make that explicit.